mrp_subcontracting_partner_management/models/res_partner.py

```diff
@@ -6,7 +6,7 @@
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
-    is_subcontractor_partner = fields.Boolean(string="Subcontractor")
+    is_subcontractor_partner = fields.Boolean(string="Is subcontractor")
```

Comment:
Thinking twice, and following how is the rest of the module, this seems to be `Subcontractor partner`.

Reply:
Ok, do I change this also in v14?

Reply:
As you prefer